cmd/snap/cmd_snapshot.go

```diff
@@ -410,9 +417,22 @@ func init() {
 				desc: i18n.G("The filename of the export"),
 			},
 		})
-	// This command is hidden because there's no corresponding
-	// "import-snapshot" to consume the produced data.
-	// TODO: implement import-snapshot and remove the hidden attribute.
+	// XXX: this command is hidden because import/export is not complete
+	cmd.hidden = true
+
+	cmd = addCommand("import-snapshot",
+		shortImportSnapshotHelp,
+		longImportSnapshotHelp,
+		func() flags.Commander {
+			return &importSnapshotCmd{}
+		}, nil, []argDesc{
+			{
+				name: "<filename>",
+				// TRANSLATORS: This should not start with a lowercase letter.
+				desc: i18n.G("Name of the snapshot set file to import"),
```

Comment:
what is a "snapshot set file" though ?

I understand my formulation was clunky, but this one is in a terminology nowhere. Export itself has this:

"<filename>: The filename of the export"

Possibilities could be:    " Name of the snapshot export file to import"  or "Name of the snapshot export file to use" or "Name of snapshot export file to ingest"

Reply:
Thank you @pedronis! I like "The filename of the export" and will update this.